processing/src/main/java/org/apache/carbondata/processing/loading/model/CarbonLoadModel.java

```diff
@@ -172,6 +173,16 @@
 
   private boolean isAggLoadRequest;
 
+  private CompactionType compactionType = CompactionType.NONE;
```

Comment:
It is strange to have a compactionType in load model, why not put it in compaction model

Reply:
CompactionModel is not available in the LoadCommand while LoadModel is available in both compation and loading commands

Reply:
i dont think for one parameter it is required to create a new compaction model

Reply:
This is added only for compaction, but load model is not for compaction. It is better to refactor it.

Reply:
Please add this compactionType to operationContext, don't add here